src/App.tsx

```diff
@@ -1,15 +1,35 @@
-import React from 'react';
+import React, { ChangeEvent, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { DropdownMenu } from './components/DropdownMenu';
```

Comment:
Ensure that the `DropdownMenu` component is correctly implemented and imported. Verify that it accepts the `people` and `onSelected` props as expected. If the `DropdownMenu` component is not implemented correctly, it might cause runtime errors.